src/main/java/com/develhope/spring/vehicles/service/VehicleService.java

```diff
@@ -47,5 +48,8 @@ public boolean existsById(Long id) {
         return vehicleRepository.existsById(id);
     }
 
+    public Collection<Integer> mostPurchasedVehicles () {
+        return  vehicleRepository.findMostPurchasedVehicleModel();
```

Comment:
ma funziona questa sintassi?

Reply:
si funziona, ma in realtà questa è la versione vecchia del metodo, adesso mi ritorna una collection di 
 mostPurchasedModel